common/types/block.go

```diff
@@ -37,7 +37,7 @@ func (id BlockID) AsHash32() Hash32 {
 	return Hash20(id).ToHash32()
 }
 
-var layersPerEpoch int32
+var layersPerEpoch int32 = 1
```

Comment:
why do you set a value here? it is better have a test panic than initialising to an arbitrary number and causing unexpected results in tests

Reply:
Some tests don't care about this, but since I print the epoch number in the log in more places they now panic.

I first tried to just set this to something in the test, but this interfered with other tests and I was too lazy to figure out what's going on. If you strongly believe this is a bad idea I can invest more time in it - I just didn't feel it's worth it.

Reply:
My gut feeling here is, if we have a test that asserts that this is 1 by default, and that it gets set to something else where we expect it to change, then it's fine, but I defer to you two

Reply:
Figured it out: `TestForEachInView_InMem_WithLimitedLayer` was passing only because `EffectiveGenesis` was zero. Once I called `SetLayersPerEpoch` it got set to some other value, failing the test.

I updated the expected value of the test so it now passes with `LayersPerEpoch=3`.